src/utils/verifyEDLJwt.js

```diff
@@ -29,14 +30,20 @@ export const verifyEDLJwt = async (header) => {
   // Decrypt the 'n' key in the JWK to get the public key used for verification of JWT token
   const pubKey = signKey.getPublicKey()
 
-  const decodedToken = jwt.verify(token, pubKey, { algorithms: ['RS256'] }, (err, decoded) => {
-    if (err) {
+  let decodedToken = {}
+
+  try {
+    decodedToken = jwt.verify(token, pubKey, { algorithms: ['RS256'] })
+    console.log('JWT Token validated successfully')
+  } catch (err) {
```

Comment:
These multiple console logs don't seem super useful, same with the valid log above. Why not just log out the error object? (if it is an object call `JSON.stringify(err)` to get a useful log message)